src/jvm/test/org/jetbrains/kotlinx/lincheck_test/transformation/KotlinStdlibTransformationTest.kt

```diff
@@ -54,5 +54,6 @@ class KotlinStdlibTransformationTest : AbstractLincheckTest() {
 
     override fun <O : Options<O, *>> O.customize() {
         iterations(1)
+        invocationTimeout(20_000)
```

Comment:
Why do you need this change?

Reply:
That was a performance "bug" actually, already fixed it and reverted the change.